source/renderer/app/components/wallet/paper-wallet-certificate/InstructionsDialog.js

```diff
@@ -169,6 +171,7 @@ export default class InstructionsDialog extends Component<Props> {
         aria-hidden
       >
         {intl.formatMessage(messages.cardanoExplorer)}
+        <SVGInline svg={externalLinkIcon} />
```

Comment:
Same, shouldn't it be underlined only on hovering?

Reply:
cc @a-rukin 

![image](https://user-images.githubusercontent.com/1504716/60741766-183f4100-9f41-11e9-804d-67962c1bc4af.png)